<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>321</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="displayTreeButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>display</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>690</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>291</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>320</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>400</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>220</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90000000</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>250</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>280</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>230</y>
      <width>139</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="insert">
       <property name="text">
        <string>insert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del">
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="find">
       <property name="text">
        <string>find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="destroy">
       <property name="text">
        <string>destroy tree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_InOrder">
       <property name="text">
        <string>InOrder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_PostOrder">
       <property name="text">
        <string>PostOrder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_PreOrder">
       <property name="text">
        <string>PreOrder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteRightBranch">
       <property name="text">
        <string>destroyBranchRight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteLeftBranch">
       <property name="text">
        <string>destroyBranchLeft</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="destroySubTree">
       <property name="text">
        <string>destroySubTree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findCommonKey">
       <property name="text">
        <string>findCommonKey</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>520</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>560</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>610</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_7">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>650</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_8">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>650</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
